feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SearchViewModel.kt

```diff
@@ -99,15 +99,16 @@ class SearchViewModel @Inject constructor(
         )
     }
 
-    fun onDaySelected(day: DroidKaigi2022Day, isDeselected: Boolean) {
+    fun onDaySelected(day: DroidKaigi2022Day, isSelected: Boolean) {
+        val selectedDays = filters.value.days.toMutableList()
         filters.value = filters.value.copy(
-            day = if (isDeselected) {
-                null
-            } else {
-                day
-            }
+            days = selectedDays.apply {
+                if (isSelected)
+                    add(day)
+                else
+                    remove(day)
+            }.sortedBy(DroidKaigi2022Day::start)
```

Comment:
sorting list here to display filtered days as sorted.
![Screen Shot 2022-09-24 at 19 19 36](https://user-images.githubusercontent.com/74723074/192092741-fa6182b4-611a-4abd-9d7b-1cc7f617a027.png)